README.md

```diff
@@ -1,59 +1,34 @@
+[![License](https://img.shields.io/npm/l/@nrwl/schematics.png)](https://opensource.org/licenses/MIT)
 
-We realized that most developers don’t need more sophisticated commands — they need a more approachable way to use what the Angular CLI already does. And that’s what Angular Console is.
+# What is Angular Console?
 
+Angular CLI transformed the Angular ecosystem. We no longer have to spend time maintaining our webpack configurations, figuring out how to make tests run, fixing source maps — the CLI does this for us. It also helps us enforce consistent development practices by generating components, services, and state management modules. Since the CLI does so much, it can be intimidating and hard to learn. Most developers only use a fraction of the CLI's capabilities.
+**Angular Console is the user interface for the Angular CLI**.
+At [Nrwl](https://nrwl.io/) we realized that most developers don't need more sophisticated commands — they need a more approachable way to use what the Angular CLI already does. And that's what Angular Console is.
 Using Angular Console you can create new projects, install extensions, generate artifacts, build, test, and deploy your projects. Anything you can do with the Angular CLI, you can do with Angular Console. It also gives you a visual overview of your projects.
 
-### You can download Angular Console at [angularconsole.com](https://angularconsole.com/). We provide distributions for Windows, Mac, and Linux.
+### You can download Angular Console at [angularconsole.com](https://angularconsole.com/).
 
-# Who is it for
-
-## Beginner Friendly
-
-We aim to make Angular Console a great tool for developers who are new to Angular or Angular CLI. You can create projects, interact with your editor, run generators and commands, install extensions without ever touching the terminal or having to install any node packages globally. If you get a new laptop, you can install Angular Console and start building Angular apps. Also, Angular Console highlights the properties you are likely to use for build-in generators and commands . So if you haven’t used the CLI, you don’t get overwhelmed.
+We provide distributions for Windows, Mac, Linux, and the VS Code plugin.
 
 ## Useful to Experts
 
-At the same time, Angular Console is a robust tool that can do everything the Angular CLI can do — it’s the UI for the CLI. Every command you can run in the terminal, you can run via Angular Console. Except you no longer have to remember all the flags, names, or paths — Angular Console will help you by providing autocompletion and validating your inputs.
+At the same time, Angular Console is a robust tool that can do everything the Angular CLI can do — it's the UI for the CLI. Every command you can run in the terminal, you can run via Angular Console. Except you no longer have to remember all the flags, names, or paths — Angular Console will help you by providing autocompletion and validating your inputs.
```

Comment:
Move "at the same time" from this paragaph to the next one

Reply:
Done!